lib/network/modules/components/edges/util/EdgeBase.js

```diff
@@ -1,7 +1,6 @@
 let util = require("../../../../../util");
 let EndPoints = require("./EndPoints").default;
 
-
```

Comment:
@yoavdamri Could you please only change the lines of code that really needs to be changed. Syntax "improvements" should not be mixed with other pull requests. The only change in this file should be the `drawBackground` and the `setStrokeDashed` functions.
I know this was probably your editor that did those changes but maybe you can teach it not to this!? Thanks